app/api/base.rb

```diff
@@ -0,0 +1,14 @@
+require 'grape/active_model_serializers'
+require './app/api/v1/base'
+
+module API
+  class Base < Grape::API
+    include Grape::ActiveModelSerializers
+
+    prefix :api
```

Comment:
Not feeling great about the prefix. Was there a specific need? I don't envision ingest ever having an UI of its own really.